project.clj

```diff
@@ -48,6 +48,8 @@
              ]
             [lein-environ "1.1.0"]]
 
+  :jvm-opts ["--add-modules" "java.xml.bind"]
+
```

Comment:
I'm running java 9.  This change fixes the following error:

`Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:466)
	at clojure.lang.DynamicClassLoader.findClass(DynamicClassLoader.java:69)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:563)
	at clojure.lang.DynamicClassLoader.loadClass(DynamicClassLoader.java:77)
`